src/test/java/org/folio/auth/authtokenmodule/MainVerticleTest.java

```diff
@@ -1,21 +1,21 @@
 package org.folio.auth.authtokenmodule;
 
-import static org.hamcrest.CoreMatchers.containsString;
-import static org.junit.Assert.assertThat;
+import com.nimbusds.jose.JOSEException;
 
 import org.apache.logging.log4j.Level;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
-import com.nimbusds.jose.JOSEException;
-
 import io.vertx.core.Vertx;
 import io.vertx.ext.unit.TestContext;
 import io.vertx.ext.unit.junit.VertxUnitRunner;
 
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.MatcherAssert.assertThat;
+
```

Comment:
Static imports usually go before non-static imports:
https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing